tests/common/devices.py

```diff
@@ -1585,10 +1589,43 @@ def config_facts(self, *module_args, **complex_args):
         """
         if 'host' not in complex_args:
             complex_args['host'] = self.sonichost.hostname
-        if self.sonichost.facts['num_asic'] != 1:
-            complex_args['namespace'] = 'asic{}'.format(self.asic_index)
+        if self.sonichost.is_multi_asic:
```

Comment:
is this check required. Wont "complex_args['namespace'] = DEFAULT_NAMESPACE" work for config_facts? May be config_facts should be updated to handle DEFAULT_NAMESPACE